File: vapor-app/src/main/java/com/tevinjeffrey/vapor/ui/files/fragments/views/BottomSheetView.java
package com.tevinjeffrey.vapor.ui.files.fragments.views;

import com.tevinjeffrey.vapor.okcloudapp.model.CloudAppItem;

public interface BottomSheetView {
    void showLoading(boolean isLoading);
    void showError(String message);
    void rename(CloudAppItem cloudAppItem);
    void deleteItem(CloudAppItem cloudAppItem);
    void hideSheet();
}

File: vapor-app/src/main/java/com/tevinjeffrey/vapor/ui/files/BottomSheetView.kt
package com.tevinjeffrey.vapor.ui.files

import com.tevinjeffrey.vapor.okcloudapp.model.CloudAppItem

interface BottomSheetView {
    fun showLoading(isLoading: Boolean)
    fun showError(message: String)
    fun rename(cloudAppItem: CloudAppItem)
    fun deleteItem(cloudAppItem: CloudAppItem)
    fun hideSheet()
}
